README.md

```diff
@@ -1 +1,49 @@
-# Hello-Microverse
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse First Hello README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
```

Comment:
♻️ Great! you are using the recommended template. However, I suggest you keep all the sections marked as required on the template. The only optional one is FAQ

- [ ] Please update the README file according to the template. You should keep all the sections.